app/code/Magento/Checkout/view/frontend/web/js/view/summary/abstract-total.js

```diff
@@ -21,6 +21,30 @@ define([
             return priceUtils.formatPrice(price, quote.getPriceFormat());
         },
 
+        /**
+         * @param {*} amount
+         * @return {*|String}
+         */
+        getFormattedPercent: function (amount) {
+            var format = Object.assign({}, quote.getPriceFormat());
+
+            format.requiredPrecision = this.countPrecision(amount, format.decimalSymbol);
```

Comment:
Hello @engcom-Charlie,
I'm curious about why did you have to calculate the required precision? 

I see that the `priceUtils.formatPrice()` function already has a fallback value (precision of 2 digits) in case `format.requiredPrecision` is not defined.

Thank you